Prose/ProseLib/Sources/SidebarFeature/Footer/Footer.swift

```diff
@@ -126,12 +129,7 @@ public enum FooterAction: Equatable {
         }
 
         static var previews: some View {
-            let state = (
-                FooterState(
-                    avatar: .init(avatar: .init(url: PreviewAsset.Avatars.valerian.customURL))
-                ),
-                UserCredentials(jid: "valerian@crisp.chat")
-            )
+            let state = FooterState(credentials: "valerian@crisp.chat")
```

Comment:
Couldn't `credentials` be renamed to `jid`?
I know you talked about `UserInfo` maybe, so could this be renamed to `userInfo`?
I find `credentials` being badly named, as it doesn't contain private data (but that's probably just me and my French translation).

Reply:
I'm not touching this until we have proper data. If you look at the initializer of `FooterState` you'll recognize that there is work to be done.